http/jsonapi/generator/internal/articles/open-api_test.go

```diff
@@ -45,12 +45,15 @@ type MapTypeNumber map[string]float32
 
 // MapTypeString ...
 type MapTypeString map[string]string
+type AuthorizationBackend interface {
+	Init()
+}
```

Comment:
If there is no authentication this doesn't serve any purpose. So I'll suggest to remove `AuthorizationBackend` and `RouterWithAuthentication()` and keep `Router()` as it was.